md/monitoring.md

```diff
@@ -6,5 +6,6 @@ The Operation managment living application contains pages to monitor cases given
 
 The view is filtered by process and optionally its version.
 It is composed by a pie chart that shows the ratio between cases late, predicted late and on time.
-It also contains a list of cases that displays the success rate percentage and the time remaining before the target duration.
-You can click on a particular case to display its overview.
+It also contains a list of cases that displays the percentile of cases that took the same path than the case and finished within the target defined, as well as the time remaining before the target duration.
```

Comment:
The string to replace was "success rate threshold", in the page "Configuration". But I think it's good to try and clarify here too.
As agreed last start of sprint, we don't want to tell too much of how the percentile is computed in the documentation or the product.
So I'd say (it's a suggestion):
 "It also contains a list of cases that displays  the chances for each case to reach the target duration, according to its execution so far and the time remaining before the target duration. This remaining time is also displayed for each case."